maketest.sh

```diff
@@ -20,6 +20,10 @@ case `uname` in
     ;;
 esac
 
+if [ "$OSTYPE" = "cygwin" ]; then
```

Comment:
should that be `==` or `eq` to avoid assignment confusion?

Reply:
`eq` is numeric so inappropriate.
Could be `==` but I prefer to use the more generic POSIX standard forms wherever possible.